<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_VideoStreamLoad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>501</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Тип видеопотока</string>
    </property>
    <widget class="QRadioButton" name="radioButton_Camera">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Камера</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_VideoFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Видеофайл</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_ImageSequence">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>68</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Последовательность изображений</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>98</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Путь:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Path">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>95</y>
       <width>351</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Open">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>95</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Открыть</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_LoadVideoStream">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>125</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Загрузить видеопоток</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
